code/game/objects/structures/crates_lockers/closets/secure/kitchen.dm

```diff
@@ -0,0 +1,158 @@
+	icon_off = "fridge1"
+
+/obj/structure/closet/secure_closet/fridge/update_icon()
+	if(broken)
+		icon_state = icon_broken
+	else
+		if(!opened)
+			if(locked)
+				icon_state = icon_locked
+			else
+				icon_state = icon_closed
+		else
+			icon_state = icon_opened
+
+// for almayer.
+
+// Kitchen preparation room small quantity out of boxes...
+
+//standart organic storage.
+
+/obj/structure/closet/secure_closet/fridge/organic
+	name = "Organic"
+
+/obj/structure/closet/secure_closet/fridge/organic/Initialize()
+	. = ..()
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/apple(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/cabbage(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/carrot(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/mushroom/chanterelle(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/chili(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/corn(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/eggplant(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/potato(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/tomato(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/whitebeet(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/cherries(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/lime(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/lemon(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/orange(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/snacks/grown/banana(src)
+
+//DRy
+
+/obj/structure/closet/secure_closet/fridge/dry
+	name = "dry"
+
+/obj/structure/closet/secure_closet/fridge/dry/Initialize()
+	. = ..()
+	for(var/i = 0, i < 6, i++)
+		new /obj/item/reagent_container/food/snacks/flour(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/condiment/sugar(src)
+
+//grocery
+
+/obj/structure/closet/secure_closet/fridge/groceries
+	name = "Groceries"
+
+/obj/structure/closet/secure_closet/fridge/groceries/Initialize()
+	. = ..()
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/drinks/milk(src)
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_container/food/drinks/soymilk(src)
+	new /obj/item/storage/fancy/egg_box(src)
+	new /obj/item/reagent_container/food/condiment/enzyme(src)
+
+// Kitchen Reserve big quantity stored in boxes
+
+//meat surplus
+/obj/structure/closet/secure_closet/fridge/meat/stock
+	name = "meat"
+
+/obj/structure/closet/secure_closet/fridge/meat/stock/Initialize()
+	. = ..()
+	for(var/i = 0, i < 2, i++)
```

Comment:
Here.

Reply:
not sure what's to fix there

Reply:
isn't this format ok?
 	                            for(var/i = 0, i < 2, i++)
	                            new /obj/item/storage/box/fish(src)

Reply:
Technically fine but if you can please do it like the ones you did before in `boxes.dm` then it's all the same using the preferred syntax. `for(var/i in 1 to n)`

https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STANDARDS.md#in-to-for-loops

Reply:
change to the correct for(var/i in 1 to 7) type should be fine